src/index.css

```diff
@@ -1,4 +1,15 @@
 @import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap");
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
+  html[data-theme="isomer-next"] {
+    font-family: "Inter", system-ui, sans-serif;
+  }
+
+  html[data-theme="isomer-classic"] {
+    font-family: "Lato", system-ui, sans-serif;
+  }
+}
```

Comment:
since we have this, should we remove the `fontFamily` setting from tailwindconfig?

Reply:
Yeah good point, removed in efe45a1b1c5f3b6c071bcc7a3a870be4cb85309e.